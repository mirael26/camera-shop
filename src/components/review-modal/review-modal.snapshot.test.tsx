import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import ReviewModal from './review-modal';

jest.mock('./review-modal-form/review-modal-form', () => 'ReviewModalForm');

test('ReviewModal renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<ReviewModal/>);

  expect(asFragment()).toMatchSnapshot();
});
