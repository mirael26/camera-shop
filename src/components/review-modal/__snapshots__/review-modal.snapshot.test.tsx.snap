// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewModal renders correctly 1`] = `
<DocumentFragment>
  <div
    class="modal__content"
  >
    <reviewmodalform />
  </div>
</DocumentFragment>
`;
